file_organizer_v2/src/file_organizer/services/deduplication/backup.py

```
@@ -0,0 +1,302 @@
+        Get statistics about the backup system.
+
+        Returns:
+            Dictionary containing backup statistics
+        """
+        manifest = self._load_manifest()
+
+        total_backups = len(manifest)
+        total_size = 0
+        existing_backups = 0
+
+        for backup_key, metadata in manifest.items():
```

Ruff (0.14.13)

Loop control variable `metadata` not used within loop body

Rename unused `metadata` to `_metadata`

(B007)